spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -498,7 +512,7 @@
         it_behaves_like(
           'enrollment encountering an exception',
           exception_class: 'Faraday::ParsingError',
-          exception_message: "809: unexpected token at 'invalid'",
+          exception_message: "unexpected token at 'invalid'",
```

Comment:
You don't have to change the application logic to make this work. You can use a regex here and update the `have_logged_event` call above to use `include`.

```suggestion
          exception_message: /unexpected token at 'invalid'$/,
```

Above:
```ruby
    expect(job_analytics).to have_logged_event(
      'GetUspsProofingResultsJob: Exception raised',
      include(
        reason: reason,
        enrollment_id: pending_enrollment.id,
        enrollment_code: pending_enrollment.enrollment_code,
        exception_class: exception_class,
        exception_message: exception_message,
      )
    )
```

Reply:
Great refactor, ty!